.yarnrc

```diff
@@ -0,0 +1,2 @@
+version-tag-prefix spruce/v
```

Comment:
(comment placement is random) I think you'll want to use the Variant Regex + Task Regex combination instead of the Variant Tags + Task Tags combination! I don't think we use tags in our yaml

Reply:
Thank you, great catch! I made this update in Spruce project settings.